src/Visitors/SetPropertyValue.php

```diff
@@ -68,4 +57,17 @@ protected function makeArrayValue(array $values): Array_
 
         return new Array_($items);
     }
+
+    protected function isModifyNode(Node $node): bool
+    {
+        return $node instanceof Property && $node->props[0]->name->name === $this->name;
+    }
+
+    protected function NodeModificationProcess(Node $node): void
+    {
+        list($value, $type) = $this->getPropertyValue($this->value);
+
+        $node->props[0] = new PropertyItem($this->name, $value);
+        $node->type = new Identifier($type);
+    }
```

Comment:
```suggestion

    protected function isModifyNode(Node $node): bool
    {
        return $node instanceof Property && $node->props[0]->name->name === $this->name;
    }

    protected function nodeModificationProcess(Node $node): void
    {
        list($value, $type) = $this->getPropertyValue($this->value);

        $node->props[0] = new PropertyItem($this->name, $value);
        $node->type = new Identifier($type);
    }
```
Please move this block to the top position in the class. This logic excuted first in class. This move help us read easy this class in the future